Disable test_gpu_speed_mnist.sh perf test

mrshenli opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#27816 Disable test_gpu_speed_mnist.sh perf test**

Files test_gpu_speed_mnist.sh and test_cpu_speed_mnist.sh run
"conda install -c pytorch torchvision", which would install
pytorch-1.3 libs. These tests then work on Python source files
from master, which is inconsistent with underlying libs.

pytorch_short_perf_test_gpu is failing on master, see: https://circleci.com/gh/pytorch/pytorch/3177578?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

The error is:

```
Oct 12 02:34:34 Traceback (most recent call last):
Oct 12 02:34:34   File "main.py", line 3, in <module>
Oct 12 02:34:34     import torch
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/__init__.py", line 280, in <module>
Oct 12 02:34:34     from .functional import *
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/functional.py", line 2, in <module>
Oct 12 02:34:34     import torch.nn.functional as F
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/__init__.py", line 3, in <module>
Oct 12 02:34:34     from .parallel import DataParallel  # noqa: F401
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/__init__.py", line 5, in <module>
Oct 12 02:34:34     from .distributed import DistributedDataParallel
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/distributed.py", line 8, in <module>
Oct 12 02:34:34     import torch.distributed as dist
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/__init__.py", line 23, in <module>
Oct 12 02:34:34     from .rpc import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/__init__.py", line 9, in <module>
Oct 12 02:34:34     from .api import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/api.py", line 2, in <module>
Oct 12 02:34:34     from torch.distributed import invoke_remote_builtin, invoke_remote_python_udf
Oct 12 02:34:34 ImportError: cannot import name 'invoke_remote_python_udf'
```

Differential Revision: [D17896802](https://our.internmc.facebook.com/intern/diff/D17896802)

mrshenli committed:
Disable test_gpu_speed_mnist.sh perf test

Files test_gpu_speed_mnist.sh and test_cpu_speed_mnist.sh run
"conda install -c pytorch torchvision", which would install
pytorch-1.3 libs. These tests then work on Python source files
from master, which is inconsistent with underlying libs.

[ghstack-poisoned]

mrshenli committed:
Update on "Disable test_gpu_speed_mnist.sh perf test"


Files test_gpu_speed_mnist.sh and test_cpu_speed_mnist.sh run
"conda install -c pytorch torchvision", which would install
pytorch-1.3 libs. These tests then work on Python source files
from master, which is inconsistent with underlying libs.

[ghstack-poisoned]

rohan-varma commented:
FWIW, @yf225 has a PR to disable `short_perf_test_gpu` entirely: https://github.com/pytorch/pytorch/pull/27797

mrshenli committed:
Update on "Disable test_gpu_speed_mnist.sh perf test"


Files test_gpu_speed_mnist.sh and test_cpu_speed_mnist.sh run
"conda install -c pytorch torchvision", which would install
pytorch-1.3 libs. These tests then work on Python source files
from master, which is inconsistent with underlying libs.

pytorch_short_perf_test_gpu is failing on master, see: https://circleci.com/gh/pytorch/pytorch/3177578?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

The error is:

```
Oct 12 02:34:34 Traceback (most recent call last):
Oct 12 02:34:34   File "main.py", line 3, in <module>
Oct 12 02:34:34     import torch
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/__init__.py", line 280, in <module>
Oct 12 02:34:34     from .functional import *
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/functional.py", line 2, in <module>
Oct 12 02:34:34     import torch.nn.functional as F
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/__init__.py", line 3, in <module>
Oct 12 02:34:34     from .parallel import DataParallel  # noqa: F401
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/__init__.py", line 5, in <module>
Oct 12 02:34:34     from .distributed import DistributedDataParallel
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/distributed.py", line 8, in <module>
Oct 12 02:34:34     import torch.distributed as dist
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/__init__.py", line 23, in <module>
Oct 12 02:34:34     from .rpc import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/__init__.py", line 9, in <module>
Oct 12 02:34:34     from .api import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/api.py", line 2, in <module>
Oct 12 02:34:34     from torch.distributed import invoke_remote_builtin, invoke_remote_python_udf
Oct 12 02:34:34 ImportError: cannot import name 'invoke_remote_python_udf'
```

Differential Revision: [D17896802](https://our.internmc.facebook.com/intern/diff/D17896802)

[ghstack-poisoned]

mrshenli commented:
@pytorch retest this please

mrshenli committed:
Update on "Disable test_gpu_speed_mnist.sh perf test"


Files test_gpu_speed_mnist.sh and test_cpu_speed_mnist.sh run
"conda install -c pytorch torchvision", which would install
pytorch-1.3 libs. These tests then work on Python source files
from master, which is inconsistent with underlying libs.

pytorch_short_perf_test_gpu is failing on master, see: https://circleci.com/gh/pytorch/pytorch/3177578?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

The error is:

```
Oct 12 02:34:34 Traceback (most recent call last):
Oct 12 02:34:34   File "main.py", line 3, in <module>
Oct 12 02:34:34     import torch
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/__init__.py", line 280, in <module>
Oct 12 02:34:34     from .functional import *
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/functional.py", line 2, in <module>
Oct 12 02:34:34     import torch.nn.functional as F
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/__init__.py", line 3, in <module>
Oct 12 02:34:34     from .parallel import DataParallel  # noqa: F401
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/__init__.py", line 5, in <module>
Oct 12 02:34:34     from .distributed import DistributedDataParallel
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/nn/parallel/distributed.py", line 8, in <module>
Oct 12 02:34:34     import torch.distributed as dist
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/__init__.py", line 23, in <module>
Oct 12 02:34:34     from .rpc import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/__init__.py", line 9, in <module>
Oct 12 02:34:34     from .api import _init_rpc
Oct 12 02:34:34   File "/opt/conda/lib/python3.6/site-packages/torch/distributed/rpc/api.py", line 2, in <module>
Oct 12 02:34:34     from torch.distributed import invoke_remote_builtin, invoke_remote_python_udf
Oct 12 02:34:34 ImportError: cannot import name 'invoke_remote_python_udf'
```

Differential Revision: [D17896802](https://our.internmc.facebook.com/intern/diff/D17896802)

[ghstack-poisoned]

yf225 commented:
@mrshenli Sorry for the delay in merging https://github.com/pytorch/pytorch/pull/27797 - I just landed it which should get rid of the `pytorch_short_perf_test_gpu` CI failure. Thanks so much for your help!
